src/pages/api/statistiques/getCountContact.ts

```diff
@@ -0,0 +1,76 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import base from 'src/db/airtable';
+
+const tableNameFcuDemands = 'FCU - Utilisateurs';
+
+export const getDemands = async (): Promise<any[] | null> => {
+  const records = await base(tableNameFcuDemands)
+    .select({
+      fields: [
+        'Date demandes',
+        'Structure',
+        'Éligibilité',
+        'Ville',
+        'Code Postal',
+        'Mode de chauffage',
+        'Type de chauffage',
+        'Distance au réseau',
+        'Gestionnaire',
+        'Status',
+        'en ZDP',
+        'Conso',
+        'Logement',
```

Comment:
Do you really need to get all this fields ? 
And why don't you use the proper service ?

Reply:
No, i don't need all these fields. (It is an WIP for Florence first, to test functional value and i must clean this after his review). 
And I have not use the service because we currently need a email for controle they have an acount (but i will probably update that with anonymous sending -without firstname, lastname and email- when email is not given).